package.json

```diff
@@ -3,6 +3,12 @@
   "version": "1.4.2",
   "description": "Open color scheme for web UI",
   "main": "open-color.scss",
+  "files": [
```

Comment:
We should probably also include the scss files which are located at `docs/_sass/_open-color.scss`.
Maybe for 2.0.0 it'd be better to place all generated files at root level.